custom_components/divoom_pixoo/pages/fuel.py

```diff
@@ -16,12 +16,12 @@ def fuel(pixoo, hass, page_data: dict, FONT_PICO_8, FONT_GICKO, FIVE_PIX, ELEVEN
 
     #colors
     darkgrey = (36, 36, 36)  #datetime bg
-    black = (0, 0, 0) #default title
-    white = (255, 255, 255)  #date
-    yellow = (255, 230, 0) #default bg + time
+    black = "(0, 0, 0)" #default title
+    white = "(255, 255, 255)"  #date
+    yellow = "(255, 230, 0)" #default bg + time
 
     for key in page_data.keys():  # Convert all values to strings. Avoids problems.
-        page_data[key] = str(page_data[key])
```

Comment:
by removing this, it makes the for loop useless. You could probably remove it then..

Reply:
I don't know why, but I am firmly convinced that I have reinstalled the str() -.-